checklist_plus/analyzer/base.py

```
@@ -0,0 +1,371 @@
+
+        try:
+            # Process texts using spaCy pipe
+            docs = list(self.nlp.pipe(texts, batch_size=effective_batch_size))
+
+            # Batch process HF NER if enabled (more efficient than individual calls)
+            hf_ner_results = []
+            if include_hf_ner and hasattr(self, 'hf_ner_pipeline'):
+                try:
+                    # Process all texts at once for efficiency
+                    hf_ner_results = self.hf_ner_pipeline(texts, batch_size=effective_batch_size)
+                except Exception as e:
```

Ruff (0.14.2)

Do not catch blind exception: `Exception`

(BLE001)